moztelemetry/dataset.py

```diff
@@ -176,6 +176,9 @@ def records(self, sc, limit=None, sample=1, decode=None, summaries=None):
         if summaries and limit is None and sample != 1:
             if sample < 0 or sample > 1:
                 raise ValueError('sample must be between 0 and 1')
+            # We want this sample to be reproducible.
+            # See https://bugzilla.mozilla.org/show_bug.cgi?id=1318681
+            random.seed(42)
```

Comment:
Should we make this a parameter of the `records` function, defaulting to `42`? Then it can be overridden if one wants a different sample of the same size.